ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KVTest.scala

```diff
@@ -276,4 +257,152 @@ object KVTest {
       }
     } yield result
 
+  private[kvutils] val metrics = new Metrics(new MetricRegistry)
```

Comment:
Please move this next to the `val`s where it's being used (i.e., `keyValueCommitting`, etc.).

Reply:
Done.